api/v1alpha1/provisioning_types.go

```diff
@@ -100,12 +107,7 @@ type ProvisioningStatus struct {
 // This CR is a singleton, created by the installer and currently only
 // consumed by the cluster-baremetal-operator to bring up and update
 // containers in a metal3 cluster.
-
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:path=provisionings,scope=Cluster
-// +kubebuilder:subresource:status
-
```

Comment:
any reason for moving these back up?

Reply:
Moving this back did not affect https://github.com/openshift/cluster-baremetal-operator/blob/640297f920e13f2308d97d3b835ecf083a86cebf/config/crd/bases/metal3.io_provisionings.yaml#L16-L18.

This change was needed for the yaml file to have https://github.com/openshift/machine-api-operator/blob/master/install/0000_30_machine-api-operator_04_metal3provisioning.crd.yaml#L17-L24.